src/firefly/test/python/demo-notebook.ipynb

```diff
@@ -0,0 +1,423 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "from FireflyClient import *\n",
+    "\n",
+    "host='localhost:8080'\n",
+    "\n",
+    "def myCallback(event):\n",
+    "    print \"Event Received: \"+json.dumps(event['data']);\n",
+    "    if 'type' in event['data']:\n",
+    "        if event['data']['type']=='AREA_SELECT':\n",
+    "            print '*************area select'\n",
+    "            pParams= { 'URL' : 'http://web.ipac.caltech.edu/staff/roby/demo/wise-m51-band2.fits','ColorTable' : '9'}\n",
+    "            status= fc.showFits(fileOnServer=None, plotId='p4', additionalParams=pParams)         \n",
+    "   \n",
+    "        \n",
+    "\n",
+    "print 'HELLO HELLO HELLO'\n",
+    "fc= FireflyClient(host,'tt')\n",
+    "fc.addListener(myCallback)\n",
+    "#raw_input(\"add fits.   Press Enter to continue...\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "fc.launchBrowser()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "#file= fc.uploadFile('./data/c.fits')\n",
+    "file= fc.uploadFile('/hydra/cm/firefly/src/firefly/test/python/data/wise-1b-1.fits')\n",
```

Comment:
The user might be using standalone `firefly-exec.war` so please use a relative path: `.data/wise-1b-1.fits`